ratis-server-api/src/main/java/org/apache/ratis/server/raftlog/RaftLogSequentialOps.java

```diff
@@ -127,7 +127,7 @@ <OUTPUT, THROWABLE extends Throwable> OUTPUT runSequentially(
    * If an existing entry conflicts with a new one (same index but different terms),
    * delete the existing entry and all entries that follow it (§5.3).
    */
-  List<CompletableFuture<Long>> append(LogEntryProto... entries);
+  List<CompletableFuture<Long>> append(List<LogEntryProto> entries);
```

Comment:
Since this is in ratis-server-api, we need to deprecate the old method and add the new method.  The old method can be removed later.

```
  /**
   * Append asynchronously all the given log entries.
   * Used by the followers.
   *
   * If an existing entry conflicts with a new one (same index but different terms),
   * delete the existing entry and all entries that follow it (§5.3).
   */
  List<CompletableFuture<Long>> append(List<LogEntryProto> entries);


  /**
   * The same as append(Arrays.asList(entries)).
   *
   * @deprecated use {@link #append(List)}
   */
  @Deprecated
  default List<CompletableFuture<Long>> append(LogEntryProto... entries) {
    return append(Arrays.asList(entries));
  }
```